# Rate in runLoop()
loop_rate: 5000

# Step interval to publish in runLoop()
publish_interval: 100

# Height of xy plane marker
svm_thre: -0.1

# Limit of configuration update in one step [m], [rad]
delta_config_limit: 0.005

# Initial sample pose list
initial_sample_pose_list:
  LeftFoot:
    translation: [0.0, 0.1, 0.0]
  RightFoot:
    translation: [0.0, -0.1, 0.0]
  LeftHand:
    translation: [0.0, 0.6, 0.8]

# Number of footsteps
motion_len: 8

# Regularization weight
reg_weight: 1e-6

# Adjacent regularization weight
adjacent_reg_weight: 1e-3

# Weight for relative sagittal position between hand and foot
rel_hand_foot_weight: 1e-3

# Start foot weight
start_foot_weight: 1e3

# QP objective weight for SVM inequality error
svm_ineq_weight: 1e6

# Lower and upper limit of hand position [m]
foot_pos_limits: [[-1e20, -1e20, -1e20], [1e20, 1e20, 1e20]]

# Lower and upper limit of foot position [m]
hand_pos_limits: [[-1e20, -1e20, 0.7], [1e20, 1e20, 0.9]]

# Waist height [m]
waist_height: 0.7

# Hand lateral position [m]
hand_lateral_pos: 0.6

# Vertices of foot marker
foot_vertices: [
  [-0.07, -0.035, 0.01],
  [-0.065, -0.04, 0.01],
  [0.06, -0.04, 0.01],
  [0.085, -0.02, 0.01],
  [0.085, 0.02, 0.01],
  [0.06, 0.04, 0.01],
  [-0.065, 0.04, 0.01],
  [-0.07, 0.035, 0.01]
  ]
